js/apps/admin-ui/src/groups/components/MoveDialog.tsx

```diff
@@ -12,23 +12,12 @@ type MoveDialogProps = {
 };
 
 const moveToRoot = async (source: GroupRepresentation) => {
-  await adminClient.groups.del({ id: source.id! });
-  const { id } = await adminClient.groups.create({
-    ...source,
-    id: undefined,
-  });
-  if (source.subGroups) {
-    await Promise.all(
-      source.subGroups.map((s) =>
-        adminClient.groups.setOrCreateChild(
-          { id: id! },
-          {
-            ...s,
-            id: undefined,
-          }
-        )
-      )
-    );
+  try {
+    await adminClient.groups.create(source);
+  } catch (error: any) {
```

Comment:
We try to avoid using `any` as much as possible, it would be preferred to use [`unknown`](https://www.typescriptlang.org/docs/handbook/2/functions.html#unknown) instead. Since TypeScript defaults to this type for errors, the type can be omitted.

```suggestion
  } catch (error) {
```

Reply:
I just followed the pattern used in the related function "moveToGroup". If you would like me to fix this, should I change it in both places?

Reply:
Yeah, there are still some places we are using `any` that I want to phase out. No need to update older code under this PR, but I would like to prevent introducing new code using `any`.